src/Node/GethNode.ts

```diff
@@ -133,8 +140,9 @@ export default class GethNode extends Node {
       fs.copySync(
         path.join(
           Directory.projectRoot,
-          'utility_chains',
-          `utility_chain_${this.chain}`,
+          'chains',
```

Comment:
This logic should be extracted in Directory class. This is used at more than one place.

Reply:
I will prefer to do this in https://github.com/OpenST/mosaic-chains/issues/110
Don't want to make changes in the existing code.